src/subscription_manager/managercli.py

```diff
@@ -1637,6 +1637,11 @@ def _validate_options(self):
                     bad = True
             if bad:
                 system_exit(os.EX_USAGE)
+        elif self.options.pool_ids:
+            if not self.cp.has_capability("remove_by_pool_id"):
+                print _("Error: The remove --pool option is not supported by the version of the entitlement server to which you are currently registered."
```

Comment:
Maybe we can use simplier language: The registered entitlement server does not support removal by pool .....